pipeline/src/aggregator/mod.rs

```diff
@@ -356,7 +353,7 @@ mod tests {
         conclusion_feed: RecordBatch,
     ) -> anyhow::Result<impl std::fmt::Display> {
         let ctx = init_ctx_cont(conclusion_feed).await?;
-        let shutdown_signal = tokio::time::sleep(tokio::time::Duration::from_millis(100));
+        let shutdown_signal = tokio::time::sleep(tokio::time::Duration::from_millis(500));
```

Comment:
this is probably okay, but it might be nice to use a oneshot channel of () and just send that in at the end. if this seems reliable, i'm fine with leaving it as is.

Reply:
The continuous run loop doesn't end, so we need some kind of timeout to give it enough time to run. As I added an extra row to the test case I needed to give it more time. Alternatively poll the results until we get a passing result from the tables, I'll look into changing that.